x/tokenregistry/types/msgs.go

```diff
@@ -274,11 +274,11 @@ func (msg TokenMetadataAddRequest) ValidateBasic() error {
 	}
```

Comment:
Should we also validate NetworkDescriptor here?

Reply:
will check the NetworkDescriptor